packages/react-checkbox/package.json

```
@@ -44,18 +44,26 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
-    "@fluentui/react-compose": "^1.0.0-beta.16",
-    "@fluentui/react-hooks": "^8.0.0-beta.15",
-    "@fluentui/react-icons-mdl2": "^1.0.0-beta.17",
-    "@fluentui/set-version": "^8.0.0-beta.2",
-    "@fluentui/style-utilities": "^8.0.0-beta.19",
-    "@fluentui/utilities": "^8.0.0-beta.15",
+    "@fluentui/react-compose": "^1.0.0-alpha.0",
+    "@fluentui/react-hooks": "^8.0.0",
+    "@fluentui/react-icons-mdl2": "^1.0.0",
+    "@fluentui/set-version": "^8.0.0",
+    "@fluentui/style-utilities": "^8.0.0",
+    "@fluentui/utilities": "^8.0.0",
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
     "@types/react": ">=16.8.0 <18.0.0",
     "@types/react-dom": ">=16.8.0 <18.0.0",
     "react": ">=16.8.0 <18.0.0",
     "react-dom": ">=16.8.0 <18.0.0"
+  },
+  "beachball": {
+    "tag": "beta",
+    "disallowedChangeTypes": [
+      "major",
+      "minor",
+      "patch"
+    ]
```

```suggestion
  }
```

The same as for `@fluentui/react-cards`, this package will be used for convergence. I don't think that we should update `beachball` config for v8 release there.